src/main/java/org/jabref/gui/ai/components/summary/SummaryShowingComponent.java

```diff
@@ -32,20 +42,51 @@ public SummaryShowingComponent(Summary summary, Runnable regenerateCallback) {
 
     @FXML
     private void initialize() {
-        summaryTextArea.setText(summary.content());
+        initializeWebView();
+        updateContent(false); // Start in plain text mode
+        updateInfoText();
+    }
+
+    private void initializeWebView() {
+        contentWebView = WebViewStore.get();
+        VBox.setVgrow(contentWebView, Priority.ALWAYS);
+
+        themeManager.installCss(contentWebView.getEngine());
```

Comment:
Unfortunately, this doesn't work..

@calixtus, @koppor is there any way for browser to use JabRef's style? Font family and size specifically? Or should we leave the default font in `WebView` (I guess it's some Serif)?

Reply:
@InAnYan What do I have to do, just to remove themeManager?

Reply:
I unresolved the conversation as actions were taken before @InAnYan's response. The tagged maintainers have also not responded yet, so please don't mark resolved.